test/test_runtime.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for Runtime class."""
+import pytest
+from packaging.version import Version
+from pytest_mock import MockerFixture
+
+from ansible_compat.runtime import Runtime
+
+
+def test_runtime_version() -> None:
+    """Tests version property."""
+    runtime = Runtime()
+    version = runtime.version
+    assert isinstance(version, Version)
+    # tests that caching property value worked (coverage)
+    assert version == runtime.version
+
+
+def test_runtime_version_fail(mocker: MockerFixture) -> None:
```

Comment:
Nit: I normally advocate for using pytest's builtin `monkeypatch` for as long as you don't use things uniquely accessible only via `unittest.mock.patch`.